web/src/components/troubleshoot/SupportBundleRow.tsx

```diff
@@ -292,7 +308,7 @@ class SupportBundleRow extends React.Component {
                     statusDiv
                   ) : bundle?.analysis?.insights?.length ? (
                     <div className="flex flex1 alignItems--center">
-                      {errorInsights.length > 0 && (
+                      {errorInsights && errorInsights.length > 0 && (
```

Comment:
since we're using `undefined` as the default value instead of an empty array, need to check that it actually exists before using the length accessor